### PR TITLE
Add Security Chaos Engineering to the main Glossary - english version

### PR DESCRIPTION
**Pull request by fsbaraglia**

### CODE REVIEW

File: content/en/security_chaos_engineering.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Security Chaos Engineering
+status: Completed
+category: concept
+---
+
+## What it is
+Security Chaos Engineering or SCE is a discipline of performing proactive security experimentation on a distributed system in order to build confidence in the System's capability to withstand turbulent and malicious conditions. Security Chaos Engineers use scientific method loop (steady-state, hypothesis, continuous verification, lesson learned, implement mitigation). This approach helps SOC Engineers, [Site Reliability Engineers](/site_reliability_engineering/), [DevOps](/devops/) and [DevSecOps](/devsecops/) Engineers to create confidence on the system, increase cyber resiliency and improve Security [Observability](/observability/).
```
Comment (JasonMorgan) [inline content/en/security_chaos_engineering.md:8]: Couple questions:

- Is this practice only applicable to distributed systems?
- Do we need to define chaos engineering first?
- Does this definition also include broader operational concerns or is it only applicable to appsec?

Comment (JasonMorgan) [inline content/en/security_chaos_engineering.md:8]: As a note for style purpose, this section should be restricted to defining the practice. How it helps will go in the last section.

File: content/en/security_chaos_engineering.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Problem it addresses
+An SRE or Cyber Security Engineer's main priority is to restore the service as fast as possible. Given their nature of incidents and practices, both deal with pre-failure and post-failure situations. However, they perceive as opposing disciplines. SREs follow ultimate reliability, and Cyber Security Engineers focus on the System's solid security protection and defense strategy. Unfortunately, most security issues are challenging to discover and quickly patch, impacting the business and the correct functionality of the application or the System. Those lacks are usually tricky to find out in the development phase.
```
Comment (JasonMorgan) [inline content/en/security_chaos_engineering.md:11]: Please edit this section for clarity. Unless required please limit the problem statement to the one that SCE addresses without mentioning SRE concerns.

File: content/en/security_chaos_engineering.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+## How it helps
+Security Chaos Engineering is built around [observability](/observability/) and cyber resiliency practices, aiming to uncover the "unknown unknowns" and build confidence in the System, increase cyber resiliency and improve Security observability. Engineering teams will progressively work against missing understanding for security concerns within complex infrastructure, platforms, and distributed systems. The expected results of Security Chaos Engineering are improvements in the cyber resiliency of the whole product, uncovering hidden security issues, exposing the classical blind spots, and preparing the teams for critical edge cases.
```
Comment (JasonMorgan) [inline content/en/security_chaos_engineering.md:14]: Some of this section belongs in the "what it is" portion. "How it helps" should be limited to describing what SCE does that is beneficial and addresses the problem described in the preceding section.

**Comment by JasonMorgan** [comment]
@fsbaraglia great start but I have a lot of questions. Most are added in line but the main one being should we define chaos engineering first? It seems like this is a subset of that therefore it would benefit from a definition of CE.

I would appreciate your thoughts on this.

Thanks!

**Comment by JasonMorgan** [comment]
Converted to draft while we wait on Chaos Engineering.

**Comment by fsbaraglia** [comment]
Hi @JasonMorgan  , I am finalizing the changes to security chaos engineering, I added in the mean-while the Chaos Engineering concept, if you can please have a look. Thank you

File: content/en/security_chaos_engineering.md (hunk 1 above)
Comment (fsbaraglia) [reply content/en/security_chaos_engineering.md:8]: - yes it is mainly applicable to distributed systems, for just standalone apps, we can just run the classic regression tests, performance test etc...
- defined and linked 👍 
- yes, it includes infra-sec, app-sec, platform-sec and cloud/hybrid/multi-cloud-security

Comment (fsbaraglia) [reply content/en/security_chaos_engineering.md:8]: changes made on this paragraph

File: content/en/security_chaos_engineering.md (hunk 2 above)
Comment (fsbaraglia) [reply content/en/security_chaos_engineering.md:11]: done

**Comment by jihoon-seo** [comment]
[Spellcheck CI fails](https://github.com/cncf/glossary/runs/5270751693?check_suite_focus=true).

Adding these words to [`wordlist.txt`](https://github.com/cncf/glossary/blob/main/wordlist.txt) will make Spellcheck CI pass!
- sce
- cyber
- sres

**Comment by JasonMorgan** [comment]
Sorry @fsbaraglia I didn't realize we'd combined these two into one PR. I'll review it now for content but before we merge I'd like each term in it's own PR.

**Review by JasonMorgan — APPROVED** [review]

**Comment by JasonMorgan** [comment]
The terms look great and I'm happy to merge them in. Can you please split Chaos Engineering into it's own PR and appease the spellchecker? This is good stuff and I'm looking forward to getting it merged in.

File: content/en/chaos_engineering.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+title: Chaos Engineering
+status: Completed
+category: concept
+---
+
+## What it is
+Chaos Engineering or CE is the discipline of experimenting on a [distributed system](https://glossary.cncf.io/distributed_systems/) in production to build confidence in the system's capability to withstand turbulent and unexpected conditions.
+
+## Problem it addresses
+[SREs](https://glossary.cncf.io/site_reliability_engineering/) and [DevOps](https://glossary.cncf.io/devops/) continuously look for new techniques to increase product resiliency and [reliability](https://glossary.cncf.io/reliability/). A system's ability to tolerate failures while ensuring adequate service quality is typically a software development requirement. But deep testing on a production system is not always possible due to the complexity of the architecture. High deployment of new features to the production environment can result in a high probability of downtime and a critical incident — with considerable consequences to the business.
```
Comment (CathPag) [inline content/en/chaos_engineering.md]: Not sure I understand "high deployment of new features..."

File: content/en/security_chaos_engineering.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+title: Security Chaos Engineering
+status: Completed
+category: concept
+---
+
+## What it is
+Security Chaos Engineering or SCE is a discipline based on [Chaos Engineering](/chaos_engineering/). SCE performs proactive security experimentation on a distributed system to build confidence in the system's capability to withstand turbulent and malicious conditions. Security chaos engineers use scientific method loops to achieve this, including steady-state, hypothesis, continuous verification, lesson learned, and mitigation implementation.
+
+## Problem it addresses
+The main priority for [site reliability engineers](/site_reliability_engineering/) (SREs) and cyber security engineers is to restore service as fast as possible with the goal of achieving zero downtime and minimizing business impact. Incidents and practices deal both with pre-failure and post-failure situations. Most security issues are challenging to discover and patch quickly, impacting application or system functionality. Additionally, security incidents are usually tricky to uncover during the development phase.
```
Comment (CathPag) [inline content/en/security_chaos_engineering.md:11]: I'm struggling to understand how these sentences connect. The first sentence is clear, but how does "incidents and practices deal both with pre-failure and post-failure situations" fit in and how does it connect to the following sentence. In the last sentence, I added "additionally" because it feels like an extra point. 

I did do some edits but don't think I changed the meaning of the sentences (although I may have by mistake).

File: content/en/security_chaos_engineering.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+## How it helps
+Security Chaos Engineering is built around [observability](/observability/) and cyber resiliency practices. It aims to uncover the "unknown unknowns" and build confidence in the system, increasing cyber resiliency and improving observability.
+
+Engineering teams will progressively work against missing understanding for security concerns within complex infrastructure, platforms, and distributed systems. SCE improves the cyber resiliency of the entire product, uncovers hidden security issues, exposes the classical blind spots, and prepares teams for critical edge cases. This approach helps SREs, [DevOps](/devops/) and [DevSecOps](/devsecops/) engineers create confidence in the system, increase cyber resiliency and improve observability.
```
Comment (CathPag) [inline content/en/security_chaos_engineering.md:16]: Not sure I get "work against missing **understanding**." Do you mean "unknown security concerns"?

**Review by CathPag — COMMENTED** [review]
Looks great! I made some edits to the field directly. Please have a look to make sure you like them (and I didn't mess anything up).

I do have a few questions. Thank you!

**Comment by fsbaraglia** [comment]
> The terms look great and I'm happy to merge them in. Can you please split Chaos Engineering into it's own PR and appease the spellchecker? This is good stuff and I'm looking forward to getting it merged in.

yes, splitted and created a new PR :D

File: content/en/chaos_engineering.md (hunk 4 above)
Comment (fsbaraglia) [reply content/en/chaos_engineering.md]: added high-frequency deployment

File: content/en/security_chaos_engineering.md (hunk 6 above)
Comment (fsbaraglia) [reply content/en/security_chaos_engineering.md]: changed the wording

File: content/en/security_chaos_engineering.md (hunk 5 above)
Comment (fsbaraglia) [reply content/en/security_chaos_engineering.md]: you are right, let me rephrase it, adding "SREs and Cyber Security Engineers" as subject

**Comment by fsbaraglia** [comment]
Hey @JasonMorgan , Hey @CathPag , here the PR for Chaos Eng https://github.com/cncf/glossary/pull/498

**Comment by fsbaraglia** [comment]
nice,  great and thank you all @CathPag  @iamNoah1  @JasonMorgan

**Comment by CathPag** [comment]
The spellcheck is failing. While we wait for Chaos Engineering to be merged, can you please add the missing words, @fsbaraglia?

**Comment by fsbaraglia** [comment]
Sure, let me check if I have the right to add something to wordlist.txt, otherwise I will add to this pull request :)

**Comment by fsbaraglia** [comment]
> The spellcheck is failing. While we wait for Chaos Engineering to be merged, can you please add the missing words, @fsbaraglia?

done, added to the PR the missing words

**Comment by CathPag** [comment]
...and here goes #2 :)
